interfaces/acados_template/acados_template/c_templates_tera/acados_solver.in.h

```diff
@@ -144,6 +144,12 @@ nlp_solver_capsule * {{ model.name }}_acados_create_capsule(void);
 int {{ model.name }}_acados_free_capsule(nlp_solver_capsule *capsule);
 
 int {{ model.name }}_acados_create(nlp_solver_capsule * capsule);
+/**
+ * More generic acados_create function, since the number of stages usually is independent of the model and its
```

Comment:
I suggest changing the comment to:
"Generic version of {{ model.name }}_acados_create which allows to use a different number of shooting intervals than the number used for code generation."